### ISSUE TITLE
[Bug] Prism Project Cannot resolve type "PrismApplication". error on App.xaml Page 

### ISSUE THREAD

**Issue by mistrypragnesh40** [PrismLibrary/Prism.Templates#38]
## Summary
Hello Team,
I have created a new Xamarin Project with Prism Framework. 

When I try to build a project I am getting  this error:
XamlC error XFC0000 : Cannot resolve type "PrismApplication"

The application was running but When I try to access StaticResource on the content page that time
No Intelligence is showing due to the above error.
App.XAML file.
```
<?xml version="1.0" encoding="utf-8" ?>
<prism:PrismApplication  
             xmlns:prism="http://prismlibrary.com" 
             xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:renderer="clr-namespace:CollegeApp.Renderer"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="CollegeApp.App">
    <Application.Resources>
        <ResourceDictionary>
	    <Style TargetType="Label" x:Key="LabelHeaderTitle">
                <Setter Property="FontSize" Value="26"/>
            </Style>
         </ResourceDictionary>
    </Application.Resources>
</prism:PrismApplication>
```
App.xaml.cs
```
public partial class App : PrismApplication
{

        protected override void OnStart()
        {
        }

        protected override void OnSleep()
        {
        }

        protected override void OnResume()
        {
        }

        protected override void RegisterTypes(IContainerRegistry containerRegistry)
        {
            #region Navigation
            containerRegistry.RegisterForNavigation<NavigationPage>(Pages.Navigation);
            #endregion
        }

        public App() : this(null) { }

        public App(IPlatformInitializer initializer) : base(initializer) { }

        protected async override void OnInitialized()
        {
            InitializeComponent();

            await NavigationService.NavigateAsync(Pages.Startup.LoginPage);
        }
 }
```

Prism.Forms 8.1.97
Prism.DryIoc.Forms  8.1.97

**Comment by bkaankose** [comment]
Looks like the issue only appears when you use the source link as a xaml namespace.

If you replace
`xmlns:prism="http://prismlibrary.com" `

with (replace the container name part you use with Prism)
`xmlns:prism="clr-namespace:Prism.Unity;assembly=Prism.Unity.Forms"`

issue is gone. Not sure if it's Prism related bug and team should address this. Looks more like a Visual Studio namespace resolver issue.

**Comment by vppetrov** [comment]
I am using Prism with DryIoc (and not unity). Following @bkaankose's suggestion I replaced
`xmlns:prism="http://prismlibrary.com"`
with
`xmlns:prism="clr-namespace:Prism.DryIoc;assembly=Prism.DryIoc.Forms"`
in my App.xaml file and the error disappeared.

I had the same error with PopupPage from Rg.Plugins.Popup, so I also replaced
`xmlns:pages="http://rotorgames.com"`
with
`xmlns:pages="clr-namespace:Rg.Plugins.Popup.Pages;assembly=Rg.Plugins.Popup"`
and that error disappeared as well.

As these are two different libraries, acting similarly, it might be a general VS issue. But could also be the same error in two libraries.
